wgpu/src/backend.rs

```diff
@@ -24,10 +24,10 @@ pub struct Backend {
     quad_pipeline: quad::Pipeline,
     text_pipeline: text::Pipeline,
     triangle_pipeline: triangle::Pipeline,
+    offscreen_pipeline: offscreen::Pipeline,
```

Comment:
Could we create the `Pipeline` locally in the `screenshot` method only if necessary? How much of an overhead would that be?